### DISCUSSION TITLE
Crawler without queryselector ?

### DISCUSSION THREAD

**Discussion by c0ncentus** [projectdiscovery/katana#328] [Q&A]
hi, 
I looking the lib from newsletter and I didnt get what crawler is usefull without querystring.

I search into the doc but didnt get.

does katana crawl content from websites ? 
(mean "content":extract text from html)

**Comment by MetzinAround** [comment]
@ehsandeep question about what katana is crawling? I think. @c0ncentus, I'm having a hard time understanding your question. could you restate it?

**Comment by c0ncentus** [comment]
a crawler fetch data over multiple website.
https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector

queryselector is designed for select some html content (extracts specific content on html content).

Katana support queryselector ?
katana include queryselector features ? 



When you search a ton of data, isnt good when storing in html ...

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
